buildconfig/Jenkins/Conda/run-tests

```diff
@@ -56,13 +56,19 @@ function onexit {
 
 # Clean up prior to testing
 # Prevent race conditions when creating the user config directory
-userconfig_dir=$HOME/.mantid
-rm -fr $userconfig_dir
+if [[ $OSTYPE == "msys"* ]]; then
+    userconfig_dir=$WORKSPACE/build/bin/Release
+    rm -rf $userconfig_dir/Mantid.user.properties
+    rm -rf $userconfig_dir/Mantid.properties
```

Comment:
This one is static and created by the build so we shouldn't remove this.

Reply:
Mantid.properties shouldn't be removed ok I will do that

Reply:
Done :+1: